docs/devops/quarterly_review.md

```diff
@@ -2,7 +2,11 @@
 This is a procedure of reviewing the production code quarterly to reduce application security risks.
 
 ## Team member access review:
-Review the access privilege of team member in Github and Openshift platform. Remove or downgrade the access when the team member no longer worked with the application.
+Remove or downgrade the access when the team member no longer worked with the application.
```

Comment:
Remove or downgrade the access when a team member no longer supports an application.

Reply:
Updated. Thanks.